static/css/components/jquery.autocomplete.less

```
@@ -3,25 +3,27 @@
 /* stylelint-disable no-descending-specificity */
 .ac_results {
   padding: 0;
-  overflow: hidden;
+  margin: 0;
+  overflow: auto;
   z-index: @z-index-level-16;
   position: absolute;
   display: none;
   top: -5px;
-  width: 493px;
   max-height: 290px;
```

To fix the width issue.

```suggestion
  max-height: 290px;
  max-width: 600px;
```